nextjs_tdd/src/context/index.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+
+import React, { Dispatch, SetStateAction, createContext, useContext, useState } from 'react';
+
+type Place = {
+    place: string;
+};
+
+type PlaceContextType = {
+    selectCity: Place | undefined;
```

Comment:
I feel better to avoid undefined in type here, you can setup default value

Reply:
Updated with default values.